src/microbe_stage/SpawnSystem.cs

```diff
@@ -361,7 +363,16 @@ private void SpawnMicrobesAroundPlayer(Vector3 playerLocation, ref int spawnsLef
     }
 
     /// <summary>
-    ///   Does a single spawn with a spawner
+    ///   Checks whether we're currently blocked from spawning this type
+    /// </summary>
+    private bool SpawnsBlocked(Spawner spawnType)
+    {
+        return spawnType is not CompoundCloudSpawner &&
```

Comment:
I think a slightly cleaner thing to do would be to extend the Spawner classes to know if they spawn entities or not. And then this check would just be if the type spawns entities and the estimate is over the limit. The spawn system is purposefully written so that it wouldn't care about the concrete types of the spawners.

Reply:
I considered that but didn't think it was worth it, since there's only one spawner not spawning entities that we can easily check. But I can definitely make that change if you'd prefer. It would certainly help in case we add further spawner types.

Reply:
That's my reasoning why it should be done. Overall is just goes against the design of the spawn system if the system knows anything about the concrete spawner types.